### ISSUE TITLE
cannot share project

### ISSUE THREAD

**Issue by adrianvb** [overleaf/overleaf#686]
our sharelatex/overleaf is running as a container on docker swarm using `sharelatex/sharelatex:2.1.0`

## Steps to Reproduce
1. Create a new project
2. Click on share button and fill out form
3. Use "share" to invite someone

## Expected Behaviour
dialog window closes, user is invited

## Observed Behaviour
dialog stays open, animation icon is spinning - but nothing happens

## Technical Info
firefox console displays:

```
TypeError: "h.recaptchaDisabled is undefined"
    s https://overleaf-mews.apps.uni-hamburg.de/js/ide-fddd9e876def50dd4445.js:2
    addMembers https://overleaf-mews.apps.uni-hamburg.de/js/ide-fddd9e876def50dd4445.js:2
    u https://overleaf-mews.apps.uni-hamburg.de/js/libraries-96c7272e30460348884a.js:2
    f https://overleaf-mews.apps.uni-hamburg.de/js/libraries-96c7272e30460348884a.js:2
    n https://overleaf-mews.apps.uni-hamburg.de/js/libraries-96c7272e30460348884a.js:2
 undefined libraries-96c7272e30460348884a.js:2:531883
```
(edited out a couple of source map errors)

* Browser Name and version: Firefox 72
* Operating System and version (desktop or mobile): Windows 10
* Signed in as: admin
* Project and/or file:

[edit: markdown corrected for source map errors]

**Comment by 2014CAIS01** [comment]
I meet the same problem right now.
Chrome console displays:
```
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
```
```
angular.js:14525 TypeError: Cannot read property 'invite' of undefined
    at s (ShareProjectModalController.js:202)
    at ShareProjectModalController.js:125
    at angular.js:20178
    at f (angular.js:6274)
    at angular.js:6554 undefined
```

**Comment by ShaneKilkelly** [comment]
I think this is related to a problem which we're aware of, and are preparing a fix for in the next release cycle: https://github.com/overleaf/overleaf/issues/652

Can you try the temporary fix described in this comment and see if it helps? https://github.com/overleaf/overleaf/issues/652#issuecomment-574658899

Thanks!

**Comment by mserranom** [comment]
Managed to reproduce, will confirm once it's fixed.

**Comment by mserranom** [comment]
Most likely the same error as in: https://github.com/overleaf/overleaf/issues/684

**Comment by mserranom** [comment]
This issue should've been fixed, try running `sharelatex/sharelatex:2.1.1-beta1` image.
